Makefile

```diff
@@ -58,18 +58,15 @@ test-analysis:
 
 # Metrics.
 test-metrics:
-	ginkgo -v ./e2e/metrics/...
+	cd e2e/metrics/ && ginkgo -v --junit-report=metrics-report.xml --output-dir=${GINKGO_REPORT_DIR}
 
 # Jira Integration.
 test-jira:
-	@if [ -z "${JIRA_CLOUD_USERNAME}" ]; then echo "Error: JIRA_CLOUD_USERNAME environment variable is not defined."; exit 1; fi
-	@if [ -z "${JIRA_CLOUD_PASSWORD}" ]; then echo "Error: JIRA_CLOUD_PASSWORD environment variable is not defined."; exit 1; fi
-	@if [ -z "${JIRA_CLOUD_URL}" ]; then echo "Error: JIRA_CLOUD_URL environment variable is not defined."; exit 1; fi
-	ginkgo -v -focus "Jira cloud"
+	cd e2e/jiraintegration/ && ginkgo -v --junit-report=jiraintegration-report.xml --output-dir=${GINKGO_REPORT_DIR}
```

Comment:
Just curious, isn't `-focus` be missing there?

Reply:
Hi, no.
`-focus "<description>"` is used to run any Ginkgo spec (test) that contains the "description" as the spec's description (or part of)
Once there is a suite file (`jiraintegration_suite_test.go`) in the directory and ginkgo command is executed inside the directory or includes the directory path - any ginkgo spec within the directory will be run (unless you provide `-focus` to run a specific spec within the current suite)